web/src/components/layouts/components/header/mobile-menu.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, BoxProps, Flex, Link } from '@theme-ui/components';
+import React from 'react';
+import { SystemStyleObject } from 'theme-ui';
+import { Logo } from '../logo';
+
+const styles: SystemStyleObject = {
+  width: '100%',
+  height: '100%',
+  position: 'fixed',
+  display: 'flex',
+  justifyContent: 'flex-start',
+  alignItems: 'flex-end',
+  backgroundColor: 'primary',
+  pb: '20px',
```

Comment:
```suggestion
  pb: 4,
```

Use space matrix:

https://github.com/bjerkio/website/blob/6462179d94b57b96882b62ac10a029d2d6a5c581/web/src/gatsby-plugin-theme-ui/index.ts#L18

> The space key is a specially-named scale intended for use with margin, padding, and other layout-related CSS properties. A space scale can be defined as either a plain object or an array, but by convention an array is preferred. This is an intentional constraint that makes it difficult to add "one-off" or "in-between" sizes that could lead to unwanted and rippling affects to layout.

**Note**: If you missing a value, please ask @AnneMatilde to find you a close candidate (or choose the closest).